gamification-evm-services/src/main/java/io/meeds/gamification/evm/blockchain/BlockchainConfiguration.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * This file is part of the Meeds project (https://meeds.io/).
+ * Copyright (C) 2020 - 2024 Meeds Association contact@meeds.io
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+package io.meeds.gamification.evm.blockchain;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import org.web3j.protocol.Web3j;
+import org.web3j.protocol.http.HttpService;
+
+@Configuration
+public class BlockchainConfiguration {
+
+    @Autowired
+    private BlockchainConfigurationProperties blockchainProperties;
+
+    @Bean("polygonNetwork")
+    public Web3j getPolygonNetworkWeb3j() {
+      Web3j web3j = Web3j.build(new HttpService(blockchainProperties.getPolygonNetworkUrl()));
+      return web3j;
```

Comment:
Return value directly, no need to assign it to a variable

```suggestion
      return Web3j.build(new HttpService(blockchainProperties.getPolygonNetworkUrl()));
```